samples/bluetooth/direct_test_mode/README.rst

```diff
@@ -121,16 +121,16 @@ The behavior of the commands vary depending on the hardware configuration and Kc
 
 * DTM with front-end module support:
 
-   * The official ``0x09`` DTM command when the :kconfig:option:`CONFIG_DTM_POWER_CONTROL_AUTOMATIC` Kconfig option is enabled, which is a default state.
+   * The official ``0x09`` DTM command when the :ref:`CONFIG_DTM_POWER_CONTROL_AUTOMATIC <CONFIG_DTM_POWER_CONTROL_AUTOMATIC>` Kconfig option is enabled, which is a default state.
      This command takes into account the SoC output power and the front-end module gain to set total output power.
 
      .. note::
-        The returned output power using this command is valid only for channel 0.
+        The returned output power using this command is valid only for channel ``0``.
```

Comment:
I don't think this change is correct, since it's not talking about channel as a value for for anything.